### ISSUE TITLE
Error when loading plugins on DocPad v6.70

### ISSUE THREAD

**Issue by zenorocha** [docpad/docpad#918]
I just upgraded my DocPad project to v6.70 which supports Node v0.12 and I got this error with your plugin:

```
warning: Failed to load the plugin: docpad-plugin-rss at /myProject/node_modules/docpad-plugin-rss
The error follows:
warning: An error occured:
Error: Use CoffeeScript.register() or require the coffee-script/register module to require .coffee.md files.
  at Object._base.(anonymous function) [as .coffee] (/myProject/node_modules/docpad/node_modules/cson/node_modules/coffee-script/lib/coffee-script/coffee-script.js:210:17)
  at Module.load (module.js:355:32)
  at Function.Module._load (module.js:310:12)
  at Module.require (module.js:365:17)
  at require (module.js:384:17)
  at PluginLoader.load (/myProject/node_modules/docpad/out/lib/plugin-loader.js:131:28)
  at /myProject/node_modules/docpad/out/lib/docpad.js:2563:27
  at PluginLoader.unsupported (/myProject/node_modules/docpad/out/lib/plugin-loader.js:106:7)
  at /myProject/node_modules/docpad/out/lib/docpad.js:2547:25
  at /myProject/node_modules/docpad/out/lib/plugin-loader.js:56:18
  at /myProject/node_modules/docpad/out/lib/plugin-loader.js:90:20
  at evalmachine.<anonymous>:336:14
  at /myProject/node_modules/docpad/node_modules/safefs/node_modules/graceful-fs/graceful-fs.js:102:5
  at FSReqWrap.oncomplete (evalmachine.<anonymous>:99:15)
```

**Comment by zenorocha** [comment]
This seems to be related to an upgrade on [CoffeeScript 1.7.0](http://coffeescript.org/#1.7.0):

> When requiring CoffeeScript files in Node you must now explicitly register the compiler. This can be done with require `coffee-script/register` or `CoffeeScript.register()`. Also for configuration such as Mocha's, use coffee-script/register.

**Comment by balupton** [comment]
https://github.com/hurrymaplelad/docpad-plugin-rss/issues/8

**Comment by balupton** [comment]
I'm going to re-add explicit support for this with DocPad, as out plugin guide was outdated #924 for a very long time, so there could be a lot of plugins that are now broken.

**Comment by balupton** [comment]
Fixed in v6.71.0 February 19, 2015
